packages/itwinui-css/src/utils/component-action/component-action.scss

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+/// Visually hides an element but keeps it accessible to screen readers.
+/// Because it applies some potentially invasive styles, use this mixin on a wrapper element for best results.
+/// If it contains a focusable element, make sure to revert/exclude these styles.
+@mixin component-action {
+  all: unset;
+  text-decoration: none;
+  color: inherit;
+  user-select: text;
```

Comment:
does `user-select` do anything here?

Reply:
Yes, without it text gets selected on click.

Reply:
This might be because `iui-tile-name` has `user-select: all`. We should remove that.

Reply:
Removed select all